lib/mongo/client.rb

```diff
@@ -27,6 +28,40 @@ class Client
     # @since 2.1.0
     CRUD_OPTIONS = [ :database, :read, :write ].freeze
 
+    # Valid client options.
+    #
+    # @since 2.1.2
+    VALID_OPTIONS = [
+        :auth_mech,
```

Comment:
I'd do a 2 space indent here as it's standard formatting in Ruby, and then freeze the array.